jbpm-wb-human-tasks/jbpm-wb-human-tasks-backend/src/main/java/org/jbpm/workbench/ht/backend/server/RemoteTaskServiceImpl.java

```diff
@@ -255,10 +255,14 @@ public void deleteTaskComment(String serverTemplateId,
         UserTaskServicesClient client = getClient(serverTemplateId,
                                                   UserTaskServicesClient.class);
 
-        List<TaskComment> comments = client.getTaskCommentsByTaskId(containerId,
-                                                                    taskId);
-
-        return comments.stream().map(c -> build(c)).collect(toList());
+        try {
+            List<TaskComment> comments = client.getTaskCommentsByTaskId(containerId,
+                                                                        taskId);
+            return comments.stream().map(c -> build(c)).collect(toList());
+        } catch (KieServicesException e) {
+            // task not found
```

Comment:
Thanks for the clarification. Since this approach was already approved some time in the past, I guess it's they way to go. I was just finding it a little bit suspicious to treat an error as an empty list, from user experience perspective.
If the list can't be retrieved, it would provide incorrect information to the user that the list is empty where in reality it can contains data, but data can't be displayed.